docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -98,6 +97,43 @@ protected Object readResolve() {
         return this;
     }
 
+    protected Bind[] volumesFromString(String volumesString){
```

Comment:
I think better move single entry parsing to separate static method. Then it will be easier test parsing.